README.md

```diff
@@ -28,6 +41,32 @@ QuickStart currently provides:
 
 ...some of which have the ability for warmups and cooldowns to take place, as well as charging users.
 
-The suggested permissionsets are `quickstart.admin`, `quickstart.mod` and `quickstart.user`. More detailed documentation will appear soon.
+## Prerequisites
+* [Java] 8
+* [Git]
+
+## Cloning
+1. `git clone git@github.com:EssencePowered/Essence.git`
+2. `cd Essence`
+3. `cp scripts/pre-commit .git/hooks`
+
+## Building
+#### Notes
+1. If you do not have Gradle installed then use ./gradlew for Unix systems or Git Bash and gradlew.bat for Windows
+systems in place of any 'gradle' command.
+
+In order to build Essence, simply run the `gradle` command. You will find the compiled JAR named similar to
+`Essence-x.x.x.jar` in `builds/libs/`
+
+## Contributing
+Are you a talented programmer wanting to contribute some code? EssencePowered would love the help!
+* Read our [guidelines].
```

Comment:
I'm going to have to write a huge essay somewhere that explains how the QuickStart foundation of Essence works - because most people might be scared by all my tricks. But that's my job - I'll keep it in mind.